pkg/workloads/console/controller.go

```diff
@@ -21,24 +21,24 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	k8rec "sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	workloadsv1alpha1 "github.com/gocardless/theatre/pkg/apis/workloads/v1alpha1"
 	"github.com/gocardless/theatre/pkg/client/clientset/versioned/scheme"
-	"github.com/gocardless/theatre/pkg/logging"
+	"github.com/gocardless/theatre/pkg/reconcile"
 )
 
 const (
-	EventStart    = "reconcile.start"
-	EventComplete = "reconcile.end"
-	EventRequeued = "reconcile.requeued"
-	EventFound    = "found"
-	EventNotFound = "not_found"
-	EventCreated  = "created"
-	EventExpired  = "expired"
-	EventDeleted  = "deleted"
-	EventError    = "error"
+	EventStart    = "ReconcileStart"
+	EventComplete = "ReconcileEnd"
```

Comment:
`ReconcileEnd` -> `ReconcileComplete`? Or maybe just drop the `Reconcile` prefix from them and keep them matching the constant name? I'd imagine having 121 mappings from constant name to the thing that appears in the logs would make debugging easier

Reply:
thought you were saying it would be good to have one hundred and twenty-one mappings for a minute..